scripts/local/e2e_test.sh

```diff
@@ -9,6 +9,53 @@ TELEPORTER_PATH=$(
   cd ../.. && pwd
 )
 
+function printHelp() {
+    echo "Usage: ./scripts/local/e2e_test.sh [--component component]"
+    echo ""
+    printUsage
+}
+
+valid_components="teleporter governance validator-manager"
```

Comment:
If we just test for the existence of `./tests/local//$component/$component.test`, then we don't have to maintain this list.